backend/azure/client.go

```diff
@@ -183,7 +175,7 @@ func (a *ClientImpl) List(l vfs.Location) ([]string, error) {
 		listBlob, err := containerURL.ListBlobsHierarchySegment(ctx, marker, "/",
 			azblob.ListBlobsSegmentOptions{Prefix: utils.RemoveLeadingSlash(l.Path())})
 		if err != nil {
-			log.Fatal("Unable to list blobs.  error: " + err.Error())
+			return []string{}, err
```

Comment:
no more error log or does it get caught in the return?

Reply:
This was a defect in the original PR.  The log.Fatal would cause a panic if this error condition occurred.